modules/logging-loki-storage-aws.adoc

```diff
@@ -25,3 +25,22 @@ $ oc create secret generic logging-loki-aws \
   --from-literal=access_key_secret="<aws_access_key_secret>" \
   --from-literal=region="<aws_region_of_your_bucket>"
 ----
+
+== AWS storage for STS enabled clusters
+If your cluster has STS enabled, the Cloud Credential Operator (CCO) supports short-term authentication using AWS tokens. Secrets created by using the CCO utility (`ccoctl`) use a different format to authenticate.
+
+.Example AWS STS secret
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: <target-namespace>
+  name: <target-secret-name>
+stringData:
+  credentials: |-
+    [default]
+    sts_regional_endpoints = regional
+    role_name: <operator-role-name>
```

Comment:
This should be role_arn

Reply:
@kabirbhartiRH - for which field / value ?

Reply:
`role_name` should actually be `role_arn`